### DISCUSSION TITLE
Are others seeing this with X-windows applications?

### DISCUSSION THREAD

**Discussion by marktheminer** [microsoft/WSL#11030] [General]
Hi all,

I just found this tonight in OBS and then reproduced it with Firefox and so I think it is WSL related as OBS worked fine in WSL a few months ago.

<img width="700" alt="image" src="https://github.com/microsoft/WSL/assets/97312147/7030ebfb-30ee-4545-b9af-8a13865f088a">

<img width="646" alt="image" src="https://github.com/microsoft/WSL/assets/97312147/23a740df-1c90-4002-a65e-fb774c84c3e9">

**Comment by tcostermans** [comment]
Hi,
I encountered exactly the same problem. It doesn't happen all the time, but when it happens the work around is to shutdown wsl and start it again.
`wsl --shutdown`

Version of WSL: 2.0.14

I found this bug report in the WSLG repo:
https://github.com/microsoft/wslg/issues/1148

Kr,
Tim
